### DISCUSSION TITLE
Composer no longer symlinks bin scripts?

### DISCUSSION THREAD

**Discussion by 1ma** [composer/composer#10461] [Q&A / Support] [answered]
Composer used to create symbolic links to the executable files listed in the `bin` section of composer.json. I've just noticed that now (2.2.4) it creates copies of the files.

Is there a way to get the previous behaviour? I can't find any clue in the documentation nor the changelog.

**Comment by stof** [comment, marked as answer]
older versions of composer were *sometimes* creating symlinks and sometimes creating a proxy script (not a copy).
This logic changed in Composer 2.2 and a proxy script is now always used.

**Reply by 1ma** [reply]
Thanks. I don't know what that is, but I'll look it up :sweat_smile: Seemed like regular files to me.
